src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -3802,9 +3903,16 @@ public enum DarknessAblaze implements LogicCardInfo {
       case SPIKEMUTH_180:
       return stadium (this) {
         text "Whenever a player’s Pokémon is moved from the Active Spot to the Bench during their turn put 2 damage counters on that Pokémon."
+        def eff
         onPlay {
+          eff = delayed{
+            after FALL_BACK, {
+              directDamage 20, ef.fallenBack//, STADIUM_CARD ???
```

Comment:
I think TRAINER_CARD is still the correct source currently. Definitely don't want it to be attack at the very least. Maybe OTHER could work?

Reply:
hmm, OTHER might do. Reason I want to avoid TRAINER_CARD here is due to some abilities (I think?) and (this one definitely) ancient traits only being affected by trainers that aren't stadiums or tools.

Reply:
...However, this could be solved by adding a filter on the static barrier() method! _(which I legit wasn't aware was there in the first place)_

Reply:
Does this still need addressed? Other stadiums seem to use TRAINER_CARD currently at least, and the default is going to be ATTACK which will be blocked by many things that it shouldn't be.

Reply:
Hmm, will commit it as a TRAINER_CARD effect and try solve barrier later on.